release-process.md

```diff
@@ -39,15 +39,90 @@ If you are a new Release Manager, you can read up on the process from the follow
+   (2) DSA and Elgamal
+   (3) DSA (sign only)
+   (4) RSA (sign only)
+Your selection? 1
+RSA keys may be between 1024 and 4096 bits long.
+What keysize do you want? (2048) 4096
+Requested keysize is 4096 bits
+Please specify how long the key should be valid.
+         0 = key does not expire
+      <n>  = key expires in n days
+      <n>w = key expires in n weeks
+      <n>m = key expires in n months
+      <n>y = key expires in n years
+Key is valid for? (0) 
+Key does not expire at all
+Is this correct? (y/N) y
+
+GnuPG needs to construct a user ID to identify your key.
+
+Real name: Robert Burrell Donkin
+Email address: rdonkin@apache.org
+Comment: CODE SIGNING KEY
+You selected this USER-ID:
+    "Robert Burrell Donkin (CODE SIGNING KEY) <rdonkin@apache.org>"
+
+Change (N)ame, (C)omment, (E)mail or (O)kay/(Q)uit? O
+We need to generate a lot of random bytes. It is a good idea to perform
+some other action (type on the keyboard, move the mouse, utilize the
+disks) during the prime generation; this gives the random number
+generator a better chance to gain enough entropy.
+We need to generate a lot of random bytes. It is a good idea to perform
+some other action (type on the keyboard, move the mouse, utilize the
+disks) during the prime generation; this gives the random number
+generator a better chance to gain enough entropy.
+gpg: key 04B3B5C426A27D33 marked as ultimately trusted
+gpg: revocation certificate stored as '/home/ubuntu/.gnupg/openpgp-revocs.d/08071B1E23C8A7E2CA1E891A04B3B5C426A27D33.rev'
+public and secret key created and signed.
+
+pub   rsa4096 2021-08-19 [SC]
+      08071B1E23C8A7E2CA1E891A04B3B5C426A27D33
+uid                      Jack (test) <Jack@mail.com>
+sub   rsa4096 2021-08-19 [E]
+```
+
+Note that the last 8 digits (26A27D33) of the public key is the <a href="https://infra.apache.org/release-signing.html#key-id">key ID</a>.
 
-The last step is to update the KEYS file with your code signing key
-<a href="https://www.apache.org/dev/openpgp.html#export-public-key">https://www.apache.org/dev/openpgp.html#export-public-key</a>
+<h4>Upload Key</h4>
+
+After generating the public key, we should upload it to a <a href="https://infra.apache.org/release-signing.html#keyserver">public key server</a>.
+You can upload:
+
+either use gpg command:
+
+```
+$ gpg --keyserver keys.openpgp.org --send-key 26A27D33
+```
+
+or copy-paste the ASCII-armored public key to <a href="http://keyserver.ubuntu.com:11371/#submitKey">OpenPGP Keyserver</a>.
+The ASCII-armored public key can be generated by:
+
+```
+:::console
+$ gpg --export --armor 26A27D33
+```
+
+Please refer to <a href="https://infra.apache.org/release-signing.html#keyserver-upload">keyserver-upload</a> for details.
+
+<h4>Update KEYS file with your code signing key</h4>
+
+The code signing key is exactly the same with the ASCII-armored public key mentioned above.
+You should append it to <a href="https://dist.apache.org/repos/dist/dev/spark/KEYS">KEYS</a> by:
```

Comment:
```suggestion
You should append it to the KEYS file by:
```

It doesn't seem necessary to add url for `KEYS`. People need to run the svn command below anyway.